src/android/StatusReport.java

```diff
@@ -0,0 +1,80 @@
+package com.microsoft.cordova;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class StatusReport {
```

Comment:
put a general "what this class does" comment
```
/**
 * Represents a status report to be sent to metrics.
 */
```
(the important part is that it specifies that it's for metrics, because otherwise there's absolutely context on what these reports are for)